src/core/environment.go

```diff
@@ -580,6 +581,13 @@ func (env *EnvironmentType) Processes() (result []*Process) {
 	return processList
 }
 
+func (env *EnvironmentType) isNginxProcess(name string, cmd string) bool {
+	if name == "nginx" && !strings.Contains(cmd, "upgrade") && strings.HasPrefix(cmd, "nginx:") {
+		return true
+	}
+	return false
```

Comment:
```suggestion
	return name == "nginx" && !strings.Contains(cmd, "upgrade") && strings.HasPrefix(cmd, "nginx:")
```
You can make this a one liner